virt/virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc

```diff
@@ -0,0 +1,30 @@
+[id="virt-hot-plugging-virtual-disks"]
+= Hot-plugging virtual disks
+include::modules/virt-document-attributes.adoc[]
+:context: virt-hot-plugging-virtual-disks
+
+:FeatureName: Hot-plugging virtual disks
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+toc::[]
+
+If you _hot-plug_ a virtual disk, you can attach or detach the virtual disk to or from a virtual machine instance while a virtual machine is running.
```

Comment:
hot-plug = attach a disk to a running virtual machine
hot-unplug = detach a disk that was hot-plugged from a running virtual machine.
Your ability to do so doesn't change when you plug or unplug, its the act of attaching or detaching.

Reply:
Gotcha. OK, I cleaned this up like so:

_When you _hot-plug_ a virtual disk, you attach a virtual disk to a virtual machine instance while a virtual machine is running.

When you _hot-unplug_ a virtual disk, you detach a virtual disk from a virtual machine instance while a virtual machine is running.

Only data volumes and persistent volume claims (PVCs) can be hot-plugged and hot-unplugged. You cannot hot-plug or hot-unplug container disks._